### ISSUE TITLE
Scope of Azure AD Sharepoint access.

### ISSUE THREAD

**Issue by stevedcc** [SharePoint/sp-dev-docs#6621]
This article does not mention, that the scope of the provided access is the SharePoint REST API.  Any download links to files, that expect a user to be logged in to download, cannot be accessed using the provided link - that will result in a 401 error.  Instead, they must be transformed to be compatible with the SharePoint rest API

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e082eb56-fafb-9ef7-fadb-e0b90f5369a9
* Version Independent ID: 0f1a3528-3615-5722-2352-a1ad4622c244
* Content: [Granting access via Azure AD App-Only](https://docs.microsoft.com/en-us/sharepoint/dev/solution-guidance/security-apponly-azuread)
* Content Source: [docs/solution-guidance/security-apponly-azuread.md](https://github.com/SharePoint/sp-dev-docs/blob/master/docs/solution-guidance/security-apponly-azuread.md)
* Product: **sharepoint**
* GitHub Login: @VesaJuvonen
* Microsoft Alias: **vesaj**

**Comment by Jerry0527** [comment]
Yes, the article is using the Application permissions instead of Delegated permissions.
Application permissions is running in backend service without a signed-in user which could use to update list item data etc..
For Manipulating files, it's necessary to use delegated permissions which stand for a sign-in user.
Maybe the article need to add additional information about this.
